ncps/ncps.TESTGEN.ps1

```diff
@@ -0,0 +1,209 @@
+Param(
+    [Parameter(Mandatory=$false)] [String] $Config = "Default",
+    [Parameter(Mandatory=$true)]  [String] $DestIp,
+    [Parameter(Mandatory=$true)]  [String] $SrcIp,
+    [Parameter(Mandatory=$true)]  [ValidateScript({Test-Path $_ -PathType Container})] [String] $OutDir = "",
+    [parameter(Mandatory=$true)]  [string] $DestDir,
+    [parameter(Mandatory=$true)]  [string] $SrcDir 
+)
+
+$scriptName = $MyInvocation.MyCommand.Name
+
+#===============================================
+# Internal Functions
+#===============================================
+
+function input_display {
+    $g_path = Get-Location
+
+    Write-Output "============================================"
+    Write-Output "$g_path\$scriptName"
+    Write-Output " Inputs:"
+    Write-Output "  -Config     = $Config"
+    Write-Output "  -DestIp     = $DestIp"
+    Write-Output "  -SrcIp      = $SrcIp"
+    Write-Output "  -OutDir     = $OutDir"
+    Write-Output "============================================"
+} # input_display()
+
+function banner {
+    [CmdletBinding()]
+    Param(
+        [parameter(Mandatory=$true)] [String] $Msg

```

Comment:
Validate parameter ( [ValidateScript({-Not [String]::IsNullOrWhiteSpace($_)})] )